0089-mobile-network-onboard-fees.md

```diff
@@ -0,0 +1,39 @@
+# HIP 89: MOBILE Network Onboarding Fees 
+- Authors: [Andy Zyvoloski](https://github.com/heatedlime), [Keith Rettig](https://github.com/keithrettig), [Max Gold](https://github.com/MaxGold91)
+- Start Date: 6/14/2023
+- Category: Technical & Economic
+- Original HIP PR: [#701](https://github.com/helium/HIP/pull/701)
+- Tracking Issue: [#714](https://github.com/helium/HIP/issues/714)
+- Voting Requirements: veMOBILE
+
+## Summary
+This HIP requests that The Helium Foundation correct the MOBILE Onboarding Fee from 0 USD to 40 USD as soon as possible so that the MOBILE Onboarding Fee will be in compliance with HIP 53. 
+
+## Stakeholders
+MOBILE Hotspots - All currently onboarded MOBILE Hotspot owners. 
+
+Approved MOBILE Hotspot Makers - The sub-group of Approved Hotspot Makers that manufacture omni-protocol Hotspots that utilize the MOBILE Network and the IOT Network.  
+
+## Motivation
+Current MOBILE Hotspots, such as the FreedomFi Gateway and the Bobcat 500, are both omni-protocol Hotspots; that is, they contain both IOT and MOBILE network capabilities.  HIP 53 ('Economics Overview' section) dictates that all Hotspots must pay an Onboarding Fee of 40 USD and a Location Assert Fee of 10 USD.  [Helium Maker Ethics Documentation](https://docs.helium.com/hotspot-makers/maker-ethics/)  Section 9 indicates that Approved Hotspot Makers are responsible for funding the onboarding server for their Hotspots.
+
+At the time of onboarding these Hotspots, there was no MOBILE subDAO to which to designate the MOBILE subDAO's 40 USD Onboarding Fee.  As such, all MOBILE Hotspots were only onboarded to the IOT network with its associated Onboarding Fee of 40 USD and Location Assert Fee of 10 USD.  At the time of the migration in April of 2023, these Hotspots were incorrectly onboarded to the MOBILE network with an Onboarding Fee of 0 USD.  The error continues to this day.  As a result, 0 USD in Onboarding Fees have been burned towards the MOBILE subDAO's $A$ Factor, which introduces an error in the $A$ Score of the DAO Utility Score established in HIP 51.
```

Comment:
These hotspots were onboarded to the mobile network with an onboarding fee of 0 USD prior to the migration. The migration just kept feature parity with the old chain, which is to say no charge for mobile onboarding. 

I wouldn't call it an error. They're getting onboarded for $0, and they have 0 counting towards the A factor. It is functioning as intended, but I agree something should be changed here.

Reply:
Concur on this. So that this doesn't turn into a "he said, she said" thing, I'd ask the authors to include both perspectives and treat this HIP as a "correction for the inconsistency" with a clear path forward.

Reply:
yes  it really should have been changed when the solana Migration happened IMO

Reply:
Agree. Hindsight bias tells me we should have (as a community) tried to agree on one of the two failed HIPs.